packages/webchannel-wrapper/tools/build.js

```diff
@@ -59,31 +60,29 @@ closureBuilder.build(
     name: 'firebase.webchannel.wrapper',
     srcs: glob([resolve(__dirname, '../src/**/*.js')]),
     externs: [resolve(__dirname, '../externs/overrides.js')],
+    out: 'dist/index.closure-es.js',
+    out_source_map: 'dist/index.closure-es.js.map',
     options: {
       closure: {
+        output_wrapper:
+          '%output%\n//# sourceMappingURL=index.closure-es.js.map',
         language_out: 'ECMASCRIPT5',
         compilation_level: 'ADVANCED',
         define: closureDefines
       }
     }
   },
-  async function(errors, warnings, files, results) {
-    const filePath = resolve(__dirname, '../src/index.js');
+  async function() {
+    const filePath = resolve(__dirname, '../dist/index.closure-es.js');
```

Comment:
We probably don't want to publish intermediate files to npm.
We can either change the files array in package.json to include only the files we want to publish( it's the entire dist folder right now), or we can put the intermediate files in a tmp folder and exclude the tmp folder in .gitignore

Reply:
I used `os.tmpdir()` to put them in a temp dir outside the git repo (so we don't need to add anything to .gitignore)...  PTAL

Reply:
Nice!